tools/prowgen/pkg/testdata/simple.gen.yaml

```diff
@@ -332,6 +339,7 @@ presubmits:
           name: build-cache
           subPath: gocache
       nodeSelector:
+        kubernetes.io/arch: arm64
```

Comment:
Should this one be an amd64?

Reply:
there should be one arm one per the job config

Reply:
Correct, and I believe I am seeing one arm and one amd for the second job config. But I believe  am seeing that he first job config yielded two arm64 jobs (line 376 and line 342) (which would be incorrect).

Reply:
Good catch! was a bug, fixed in last commit